docs/tutorials/aws.md

```diff
@@ -414,6 +414,39 @@ You can configure Route53 to associate DNS records with healthchecks for automat
 
 Note: ExternalDNS does not support creating healthchecks, and assumes that `<health-check-id>` already exists.
 
+## Govcloud caveats
+
+Due to the special nature with how Route 53 runs in Govcloud, there are a few tweaks in the deployment settings.
+
+* An Environment variable with name of AWS_REGION set to either us-gov-west-1 or us-gov-east-1 is required.  Otherwise it tries to lookup a region that does not exist in govcloud and it errors out.
```

Comment:
```suggestion
* An Environment variable with name of AWS_REGION set to either us-gov-west-1 or us-gov-east-1 is required. Otherwise it tries to lookup a region that does not exist in Govcloud and it errors out.
```